src/poliastro/tests/tests_twobody/test_angles.py

```diff
@@ -7,6 +7,24 @@
 from poliastro.twobody import angles
 
 
+@pytest.fixture(scope="module")
```

Comment:
In this particular case, let's use `ANGLES_DATA` just as a module variable, without using a fixture.

Reply:
Okay!